content/en/contribute/medic/product-development-process/focused-groups.md

```diff
@@ -28,24 +30,42 @@ Key users: App Admins, Implementing Partners, App Developers, Data Scientists.
 Team members:
 - Technical Product Owner
 - Product Designer
-- UX Researcher
-- 2 Software Developers
+- UX Researchers
+- Software Developers
 - Quality Assurance Engineer
 
+Videos:
+- [Introduction to Allies Focused Working Group](https://www.youtube.com/watch?v=dsc1XMdXhXs)
+- [Inner workings of Product Team's Allies](https://www.youtube.com/watch?v=wg7rQebc9-o)
+
+See Allies OKRs [here](https://github.com/orgs/medic/projects/112/views/12).
+
 ## Infrastructure
 Focused on robust and efficient development, deployment, and support of CHT infrastructure.
 
 Key users: App Developers, App Deployers, Core Engineers.
 
 Team members:
-- 3 Site Reliability Engineers
+- Technical Product Owner
+- Site Reliability Engineers
+- Software Developers
+- Quality Assurance Engineer
+
+See Infrastructure OKRs [here](https://github.com/orgs/medic/projects/112/views/15).
 
 ## Ecosystem
-Focused on interoperability with existing digital health standards.
+This team's primary objective is to improve the Community Health Toolkit in collaboration with other members/partners in the digital health ecosystem (DHE). The team actively develops enhancements and promotes their implementation within the DHE. They engage in ongoing discovery by conducting generative interviews with key users to gain a deep understanding of the ever-evolving needs and opportunities for enhancing the CHT.
```

Comment:
```suggestion
This Focused Working Group's primary objective is to build on the Community Health Toolkit in collaboration with other members/partners in the digital health ecosystem (DHE). The team actively develops enhancements and promotes their implementation within the DHE. 
```

Reply:
Let's use FWG instead of `team` for the sake of consistency on this page.

I removed `They engage in ongoing discovery by conducting generative interviews with key users to gain a deep understanding of the ever-evolving needs and opportunities for enhancing the CHT.` because all the FWG do continuous discovery, which is inconsistent with the other FWG descriptions. We already mentioned the continuous discovery at the top of the page.
@latin-panda @Phil-Mwago

Reply:
Fixed [here](https://github.com/medic/cht-docs/pull/1219).

Reply:
Perfect, thanks!